src/main/webapp/i18n/de/modelingExercise.json

```diff
@@ -15,12 +15,12 @@
             },
             "diagramType": "Diagrammtyp",
             "problemStatement": "Aufgabenstellung:",
-            "exampleSolution": "Beispiellösung",
-            "exampleSolutionExplanation": "Erklärung Beispiellösung",
-            "createExampleSolution": "Beispiellösung erstellen",
-            "exampleSolutionRequiresExercise": "Um eine Beispiellösung zu erstellen, musst du zunächst die Erstellung der Modellierungsaufgabe abschließen.",
-            "exampleSolutionForModelingExercise": "Beispiellösung für Modellierungsaufgabe {{ exerciseTitle }}",
-            "saveExampleSolution": "Beispiellösung speichern",
+            "sampleSolution": "Beispiellösung",
+            "sampleSolutionExplanation": "Erklärung Beispiellösung",
```

Comment:
[nit] This sounds a bit weird. I would prefer "Erklärung der Beispiellösung" or "Erläuterung der Beispiellösung". But maybe we should stick with "Erklärung" as this is already used in Artemis.

Reply:
Thanks, changed it.